### DISCUSSION TITLE
whisper + Pyannote

### DISCUSSION THREAD

**Discussion by AryanSakhala** [openai/whisper#2353] [Q&A]
Can we combine speaker diarization where pyannote and whisper are both being used?
I want to have a transcription model that can differentiate speakers.
What can be best approach for that?

**Comment by jhj0517** [comment]
You can use [whisperX](https://github.com/m-bain/whisperX) for that. 

Or for the gradio app version you can use [Whisper-WebUI](https://github.com/jhj0517/Whisper-WebUI)

**Reply by AryanSakhala** [reply]
Noted, thanks
